pkg/operator/vttablet/pod.go

```diff
@@ -287,6 +288,7 @@ func UpdatePod(obj *corev1.Pod, spec *Spec) {
 	update.Volumes(&obj.Spec.Volumes, tabletVolumes.Get(spec))
 	update.Volumes(&obj.Spec.Volumes, spec.ExtraVolumes)
 	update.Tolerations(&obj.Spec.Tolerations, spec.Tolerations)
+	obj.Spec.TopologySpreadConstraints = spec.TopologySpreadConstraints
```

Comment:
Why do we overwrite here vs update (as with the preceding code)?

Reply:
It was the simplest/most straightforward way to add these - I am not aware of controllers or webhooks that would add their own `TopologySpreadConstraints` that we'd want to keep, but I can update to do a diff/merge similar to volumes/tolerations

Reply:
I guess to compare/diff `TopologySpreadConstraints` - I'd ultimately have to compare their `LabelSelectors` and I am not sure if there's a sane way to do it without some edge cases